src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger.java

```diff
@@ -292,17 +289,24 @@ private boolean hasCauseFromTheSamePullRequest(@Nullable List<Cause> causes, @Nu
 
     @Override
     public void run() {
-        if(this.getBuilder().getProject().isDisabled()) {
-            logger.info(format("Build Skip (%s).", getBuilder().getProject().getName()));
-        } else {
-            logger.info(format("Build started (%s).", getBuilder().getProject().getName()));
-            this.stashPullRequestsBuilder.run();
+        if (stashPullRequestsBuilder == null) {
+            logger.info("Not ready to run.");
```

Comment:
Shouldn't this be logged as error?

Reply:
The documentation says it's possible. I understand it happens when using pipelines. I just don't know how bad it is. I think errors should be reserved for code logic errors and really bad misconfiguration.